spinalcordtoolbox/mtsat/mtsat.py

```diff
@@ -0,0 +1,151 @@
+# coding: utf-8
+# This is the interface API to compute MTsat
+# Code is based on QMRLab: https://github.com/neuropoly/qMRLab
+# Author: Julien Cohen-Adad
+# Copyright (c) 2018 Polytechnique Montreal <www.neuro.polymtl.ca>
+# About the license: see the file LICENSE.TXT
+
+import os
+import sct_utils as sct
+from msct_image import Image
+import numpy as np
+import math
+
+def compute_mtsat(nii_mt, nii_pd, nii_t1,
+                  tr_mt, tr_pd, tr_t1,
+                  fa_mt, fa_pd, fa_t1,
+                  nii_b1map=None, verbose=1):
+    """
+    Compute MTsat and T1 map based on FLASH scans
+    Parameters
```

Comment:
Like @perone I'd advocate for the standard reST docstring style ([example](https://github.com/dadadel/pyment/blob/master/example.py.patch#L93)). If you want to use numpy or google style, you're missing new lines before the Parameters/Returns headings.